index

```diff
@@ -1,3 +1,6 @@
+76f4d39cfe2a8915c7057dacaf7c74d76a59573dc82b4c343c151824365745cd irma-demo/.idea/modules.xml
+55768f1f84fb117f1d0fc383e180b5d0aa5e5e6584c8c4a2995c15b4237f0972 irma-demo/.idea/vcs.xml
+2d16a04f0246ae5828bfad70d00b1320b88d93089a6e5ce96cbd6f7f0c96aa32 irma-demo/.idea/workspace.xml
```

Comment:
Can you delete the `.idea` files before signing? They shouldn't be included in the index.